### ISSUE TITLE
multiple nonlinear regression with keras 

### ISSUE THREAD

**Issue by songwei660131** [keras-team/keras#8966]
[seis01.txt](https://github.com/keras-team/keras/files/1603877/seis01.txt)
[XY_Time01.txt](https://github.com/keras-team/keras/files/1603878/XY_Time01.txt)

Hi, I want to use keras model to realize a multiple nonlinear regression, but the  result is not as good as what i want, the flowing is the code:
# -*- coding: utf-8 -*-
"""
Created on Sat Nov 25 00:12:02 2017

@author: Think
"""
import os
import csv
os.environ['TF_CPP_MIN_LOG_LEVEL']='2'
import tensorflow as tf
import keras
from keras.models import Sequential
from keras.layers import Dense, Activation
from keras.optimizers import SGD
from keras import regularizers
from sklearn import preprocessing
from keras.models import load_model
#from sklearn import tranform,inverse_transform

defsgd=SGD(lr=0.1)
model = Sequential()
model.add(Dense(2000, activation='relu', input_dim=505))
model.add(Dense(2000, activation='relu', input_dim=2000))
model.add(Dense(2000, activation='relu', input_dim=2000))
model.add(Dense(1001, activation='relu', input_dim=2000))
model.add(Dense(1001, activation='relu', input_dim=1001))
model.add(Dense(1001, activation='relu', input_dim=1001))
model.add(Dense(501, activation='tanh', input_dim=1001))
model.compile(optimizer='adam',loss='mean_squared_error',metrics=['accuracy'])
#model.compile(optimizer='adam',loss='mean_squared_error', metrics['mse','mae','mape','cosine'])

# Generate dummy data
import numpy as np

X=np.zeros((363,501))
csv_file=csv.reader(open('seis.dat','r'))
#print(csv_file)
kk=-1
for stu in csv_file:
    k=-1
    kk=kk+1

    for V in stu:
        k=k+1
        if k<501:X[kk,k]=float(V)
     
            
T=np.zeros((363,505))
csv_file1=csv.reader(open('XY_Time.dat','r'))
##print(csv_file)
kk=-1
for stu1 in csv_file1:
    k=-1
    kk=kk+1

    for TT in stu1:
        k=k+1
        if k<505:T[kk,k]=float(TT)
      

data_T=np.zeros((363,505))
data_T1=np.zeros((505,363))
data_T1=preprocessing.scale(T.T)
data_T=data_T1.T

data_X=np.zeros((363,501))
data_X1=np.zeros((501,363))
data_X1=preprocessing.scale(X.T)
#data_X1=scaler.transform(X.T)
data_X=data_X1.T
#data_X=X
test_data= data_T[0:150,:]
#test_data=scaler.transform(test_data)
test_labels = data_X[0:150,:]
history=model.fit(data_T[:,:], data_X[:,:], epochs=100,batch_size=1,
                  validation_data=(test_data,test_labels))
#history=model.fit(data, one_hot_labels, epochs=200,batch_size=32)

data_pre=model.predict(data_T[:,:])
#data_pre=scaler_labels.inverse_transform(data_pre)
model.save('my_seis_model.h5')   #HDF5 file
#del model # deletes the existing model
##load model
#model=load_model('my_model.h5')
import matplotlib.pyplot as plt
#list all data in history
print(history.history.keys())
#summarize history for accuracy
plt.plot(history.history['acc'])
plt.plot(history.history['val_acc'])
plt.title('model accuracy')
plt.ylabel('accuracy')
plt.xlabel('epoch')
plt.legend(['train','test'],loc='upper left')
plt.show()
#summarize history for loss
plt.plot(history.history['loss'])
plt.plot(history.history['val_loss'])
plt.title('model loss')
plt.ylabel('loss')
plt.xlabel('epoch')
plt.legend(['train','test'],loc='upper left')
plt.show()

**Comment by ucasiggcas** [comment]
Dear,
I also want to train a regressor with keras,
but there is not any one example supplied,
have you solved the question?
Could U please share your project?
Thx
